src/main.zig

```diff
@@ -49,3 +49,8 @@ pub fn main() !void {
     cow.useDefaultCow();
     try cow.say("Hello world!", .{});
 }
+
+test {
```

Comment:
What does this test do?

Reply:
I see. This is in main.zig. So it runs all tests in Cowsay. and I see you remove the run_lib_tests in the other file! Nice!

Reply:
Yep, exactly! I removed all of the `lib` in `build.zig`; a "static library" in Zig actually just means that you are exporting a **C** library, and thus must have functions that are marked as `export` to be meaningful. If you just want to make a Zig library that other Zig code will use, then `addModule` is all you need!